### DISCUSSION TITLE
Extending Saleor core - Backend

### DISCUSSION THREAD

**Discussion by Muhammedaashique** [saleor/saleor#15954] [Q&A] [answered]
I want to implement a new feature in my e-commerce application that allows vendor logins. I'm currently using Saleor for the backend. How can I extend my code to achieve this functionality? Is there any official documentation available?.

**Comment by timuric** [comment, marked as answer]
Hey @Muhammedaashique, we do not recommend extending the core, but build your own APIs instead, you can learn more at https://docs.saleor.io/docs/3.x/developer/extending/overview
